index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Mobile</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
```

Comment:
- [ ] I can see that you do not currently have a Google fonts stylesheet linked to the head section of your HTML page or on the first line of your CSS file and because of this, you don't have access to the fonts you have in your CSS. Consider going to this [link](https://fonts.google.com/) and searching for the required font based on your chosen design template and link it to the head section of your HTML page. see suggestion below,

```suggestion
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Mobile</title>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="styles.css" />
  </head>
```

Or importing it to your CSS like so:
```css
@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');

Reply:
Done.